coalib/settings/Setting.py

```diff
@@ -66,6 +66,16 @@ def __repr__(self):
     return Converter()
 
 
+def str_list():
+    '""typed_list shortcut for string""'
+    return typed_list(str)
```

Comment:
I would rather do this:
```
str_list = typed_list(str)
```
Then the user doesn't have to use redundant braces like: `str_list()`